src/components/HistoryTables/ActorState/index.tsx

```diff
@@ -99,13 +91,13 @@ export function ActorState({ address }: { address: string }) {
             <Box minWidth='200px' flex='0 1 25%'>
               State
             </Box>
-            <ViewState onClick={() => setViewActorState(!viewActorState)}>
+            <p role='button' onClick={() => setViewActorState(!viewActorState)}>
               Click to{' '}
               {viewActorState ? 'hide actor state ↑' : 'see actor state ↓'}
-            </ViewState>
+            </p>
```

Comment:
I think this might break google translator apis - the conditional rendering should conditionally render two separate `p` tags

Reply:
You are totally right, I just checked by translating to Dutch and this breaks the Google Translation indeed (it doesn't update when toggling the button). Fixed it by rendering the two separate `p`s